product-service/functions/addProduct.ts

```diff
@@ -0,0 +1,71 @@
+import { APIGatewayProxyHandler } from 'aws-lambda';
+import { Client } from 'pg';
+
+import * as config from '../serverless';
+
+const getDbOptions = () => {
+  const {
+    PG_HOST: host,
+    PG_PORT: port,
+    PG_DATABASE: database,
+    PG_USERNAME: user,
+    PG_PASSWORD: password,
+  } = (config as any).provider.environment;
+
+  return {
+    host,
+    port,
+    database,
+    user,
+    password,
+  };
+}
+
+const addProductToDb = async ({ title, description, price }) => {
+  const client = new Client(getDbOptions());
```

Comment:
Why don't you use process.enc.VAR_NAME here instead of importing it from serverless config?